src/vorta/views/diff_result.py

```diff
@@ -845,3 +852,6 @@ def data(self, index: QModelIndex, role: int = Qt.ItemDataRole.DisplayRole):
                 )
 
             return tooltip
+
+    def __init__(self, mode=None, parent=None):
+        super().__init__(mode, parent)
```

Comment:
Why is that needed?

Reply:
Yeah, with mode=None this might not be needed. Earlier when I was passing a mode i used this. Thanks for pointing this out.